### ISSUE TITLE
check for window before calling it in beamerIntialize

### ISSUE THREAD

**Issue by krisrobinson** [reenan/react-beamer#2]
Trying to use this script in a Gatsby site, but it will spit out an error when compiling server side because window is not defined. 

Error:
`WebpackError: ReferenceError: window is not defined`

Maybe a simple fix like this would work:

```jsx
 if(typeof window !== 'undefined' && window.document) {
      window.beamer_config = { product_id: id };
 }
```

**Comment by krisrobinson** [comment]
Bump... anyone here?

**Comment by reenan** [comment]
hello @krisrobinson sorry for the delayed response. I honestly don't follow this package much since the idea was to provide this very simple wrapper. 

I can for sure add this check, but if the window isn't defined, we can't set the global and thus beamer won't load correctly, I would stop the initialization at all if the window isn't there.

**Comment by reenan** [comment]
> Bump... anyone here?

should be fixed on `1.0.6`
